src/cli/cmd/ping.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from cli.config import ProjectConfiguration
+from cli.utils.yaml import write_file
+ 
+from core.common.drivers.factory import load_driver
+from core.common.drivers.factory import load_config
+from .base import BaseCmd
+
+from sqlalchemy.exc import OperationalError,DBAPIError
+class PingCmd(BaseCmd):
+  def run(self):
+      source = self.args
+      try:
+          driver_config = self.config.sources[source]
+          driver_cls = load_driver(driver_config)
+          driver = driver_cls(driver_config)
+          driver.test()
```

Comment:
This should output the result and the status of the test should be displayed accordingly